src/protocolsupport/protocol/packet/handler/EntityRewriteDownstreamBridge.java

```diff
@@ -28,4 +29,12 @@ public void handle(PacketWrapper packet) throws Exception {
 		con.sendPacket(packet);
 	}
 
+	@Override
```

Comment:
hmm ill move this

Reply:
😕

Reply:
i still cant find how/where to move this

Reply:
Can you check if this is still needed after we merge master into PSPE?